### ISSUE TITLE
Arvo doesn't have a library for converting atoms to English

### ISSUE THREAD

**Issue by keatondunsford** [urbit/arvo#683]
As discussed with @ohAitch in #682, it'd be nice to have a `/lib` file that provided this utility function of converting Hoon atom digits to their English text values.

For example, something like:

```
~zod:dojo> /+  to-english
>=
~zod:dojo> (atom:to-english 42)
"forty-two"
```

**Comment by ohAitch** [comment]
~clearly we should have a new `@en` atom format~

**Comment by keatondunsford** [comment]
_Addition_: It'd be a fun feature if this worked for arbitrarily large atoms, i.e.:

```
~zod:dojo> /+  to-english
>=
~zod:dojo> (atom:to-english 18.446.744.073.709.551.616)
"eighteen quintillion, four hundred and forty-six quadrillion, seven hundred and forty-four trillion, seventy-three billion, seven hundred and nine million, five hundred and fifty-one thousand, six hundred, and sixteen"
```

**Comment by ohAitch** [comment]
I was thinking full https://en.wikipedia.org/wiki/Names_of_large_numbers
yeah, it's not that much more code

**Comment by cgyarvin** [comment]
If my plane crashes and this goes into hoon.hoon, my angry ghost will haunt
all your repos until the end of time.

But there is a place in %clay for stuff like this.  There must be --
although the filesystem ontology still needs some work...

**Comment by Fang-** [comment]
>hoon.hoon
>%clay

I'm not sure how this would go anywhere but zuse (or maybe a `/lib` if we're super opposed to expanding the stdlib for some reason).

**Comment by ohAitch** [comment]
I'd just port https://www.npmjs.com/package/number-to-words into `lib/number-to-words.hoon`, or like wherever `userlib:zuse` ends up.

**Comment by joemfb** [comment]
resolved by #1053
